app/helpers/recommendations_helper.rb

```diff
@@ -96,12 +96,18 @@ def recommendations_sites
         link: "https://scotlandinunion.co.uk/"
       },
       "sprint-for-pr" => {
-        order: 6,
+        order: 7,
         name: "Sprint for PR",
         meta_desc: "How and where to vote tactically in the next general election " + \
                    "to advocate for proportional representation",
         link: "https://sprintforpr.uk/plan"
       },
+      "vote-climate-uk" => {
+        order: 6,
```

Comment:
Doesn't really matter, but more intuitive to have the code order match the numerical order.